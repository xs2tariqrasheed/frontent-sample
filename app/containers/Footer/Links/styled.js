import styled from 'styled-components';

export const Wrapper = styled.div`
  margin-top: 20px;
  width: 282px;
`;

export const Link = styled.a`
  opacity: 0.7;
  font-family: Roboto;
  font-size: 12px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: 2;
  letter-spacing: normal;
  text-align: right;
  color: #333333;
  text-decoration: none;
`;
