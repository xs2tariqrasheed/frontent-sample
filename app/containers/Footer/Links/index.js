/*
 * Social Media
 *
 * Social media bar
 *
 */

import React from 'react';

import { Link, Wrapper } from './styled';

export default () => (
  <Wrapper>
    <div align="right">
      <Link href="/terms-of-service">Terms of Service&nbsp;&nbsp;</Link>
      <Link href="/privacy">|&nbsp;&nbsp;Privacy Policy</Link>
    </div>
  </Wrapper>
);
