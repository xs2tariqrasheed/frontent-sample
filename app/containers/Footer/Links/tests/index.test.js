import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import Links from '../index';

describe('<Links />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <Links />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
