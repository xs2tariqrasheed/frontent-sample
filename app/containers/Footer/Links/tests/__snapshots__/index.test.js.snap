// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Links /> should render and match the snapshot 1`] = `
<div
  class="styled__Wrapper-sc-1uzqntd-0 gSbFnA"
>
  <div
    align="right"
  >
    <a
      class="styled__Link-sc-1uzqntd-1 eAFQtY"
      href="/terms-of-service"
    >
      Terms of Service  
    </a>
    <a
      class="styled__Link-sc-1uzqntd-1 eAFQtY"
      href="/privacy"
    >
      |  Privacy Policy
    </a>
  </div>
</div>
`;
